action/recipes/coreboot_test.go

```diff
@@ -66,28 +68,31 @@ func TestCoreboot(t *testing.T) {
 		filepath.Join(pwd, fmt.Sprintf("../../tests/coreboot_%s/seabios.defconfig", corebootVersion)),
 		defconfigPath,
 	)
-	// ^^^ this relative path might be funky
+	//   ^^^ this relative path might be funky
```

Comment:
One thing we could do to make it more obvious to run the tests from the correct path is to add a message to the particular asserts that check errors related to missing files. So for example something like this:
```go
assert.NoErrorf(t, err, "encountered issue with missing files, is %s the root of the repo?", currentPath)
```

Reply:
Technically speaking, this should not pose a problem to the end user, only to us. And only in case we decide to move stuff around and the relative path will no longer match the directory tree.
But I agree that having helpful error message is better than comment.